Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "timer-queue"
-version = "0.1.0"
+version = "0.2.0"
```

This is a non-breaking change.
```suggestion
version = "0.1.1"
```